lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -228,14 +225,16 @@ class CategoryRenderer {
     wrapper.href = `#${category.id}`;
     wrapper.classList.add(`lh-gauge__wrapper--${Util.calculateRating(category.score)}`);
 
+    const numericScore = Number(category.score);
     const gauge = this.dom.find('.lh-gauge', tmpl);
     // 329 is ~= 2 * Math.PI * gauge radius (53)
     // https://codepen.io/xgad/post/svg-radial-progress-meters
     // score of 50: `stroke-dasharray: 164.5 329`;
-    this.dom.find('.lh-gauge-arc', gauge).style.strokeDasharray = `${category.score * 329} 329`;
+    this.dom.find('.lh-gauge-arc', gauge).style.strokeDasharray = `${numericScore * 329} 329`;
 
-    const scoreOutOf100 = Math.round(category.score * 100);
-    this.dom.find('.lh-gauge__percentage', tmpl).textContent = scoreOutOf100;
+    const scoreOutOf100 = Math.round(numericScore * 100);
+    this.dom.find('.lh-gauge__percentage', tmpl).textContent = category.score === null ?
+      '?' : scoreOutOf100;
```

Comment:
can you also apply a title attribute to that explains why its ?

Reply:
done, text suggestion welcome right now "Errors occurred while auditing"